src/fastcat_devices/fts.cc

```diff
@@ -100,11 +100,17 @@ bool fastcat::Fts::ConfigFromYaml(YAML::Node node)
 
 bool fastcat::Fts::Read()
 {
-  for (int ii = 0; ii < FC_FTS_N_DIMS; ii++) {
-    if (!UpdateSignal(signals_[ii])) {
+  int i= 0;
+  for (auto& signal : signals_) {
+    if (!UpdateSignal(signal)) {
       ERROR("Could not extract signal");
       return false;
     }
+    MSG_DEBUG("Signal number %d is value %f", i, signal.value);
```

Comment:
This will hurt performance and not be helpful long-term when debugging full applications.

Reply:
Yes, my bad! Mostly was using this for my own purposes and thought I removed them. I will do that now. :)